ga4gh/datamodel/__init__.py

```diff
@@ -86,3 +104,47 @@ def sanitizeString(cls, attr, attrName):
         if len(attr) > cls.maxStringLength:
             attr = attr[:cls.maxStringLength]
         return attr
+
+    def _setAccessTimes(self, directoryPath):
+        """
+        Sets the creationTime and accessTime for this file system based
+        DatamodelObject. This is derived from the ctime of the specified
+        directoryPath.
+        """
+        # ctime is in seconds, and we want milliseconds since the epoch
+        ctimeInMillis = int(os.path.getctime(directoryPath) * 1000)
+        self._creationTime = ctimeInMillis
+        self._updatedTime = ctimeInMillis
+
+    def _scanDataFiles(self, dataDir, patterns):
+        """
+        Scans the specified directory for files with the specified globbing
+        pattern and calls self._addDataFile for each. Raises an EmptyDirException
```

Comment:
Dammit, I thought I'd fixed this and pushed an update.

I've been looking for a text object to properly work with python docstring comments, so I can go `gqid` (or whatever it would be) and reformat the docs. Sadly, it doesn't seem to exist.